l10n/en/firefox/faq.ftl

```diff
@@ -52,7 +52,7 @@ protecting-your-privacy = Protecting your privacy is our number one priority, an
 is-firefox-safe = Is { -brand-name-firefox } safe?
 
 # Variables:
-# $url (url) - link to https://www.mozilla.org/firefox/privacy/products/
+# $url (url) - link to https://www.mozilla.org/firefox/features/private/
```

Comment:
Silently expecting that changes to context hint comments for pontoon won't raise any issues downstream, just change the help text there when merged, nothing more happens, right?

Reply:
I believe so!